contracts/BancorX/BancorX.sol

```diff
@@ -20,7 +19,7 @@ import "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
  * Reporting cross chain transfers works similar to standard multisig contracts, meaning that multiple
  * callers are required to report a transfer before tokens are released to the target account.
  */
-contract BancorX is IBancorX, TokenHolder {
+contract BancorX is Utils, Ownable {
```

Comment:
Why `is Utils`? Why do we need to inherit and not just import?

Reply:
I am not sure it is possible to import modifiers by a library (vs a contract).